src/common/darktable.c

```diff
@@ -770,6 +770,30 @@ int dt_init(int argc, char *argv[], const gboolean init_gui, const gboolean load
     gtk_init(&argc, &argv);
   }
 
+  // execute a performance check and configuration if needed
+  int last_configure_version = dt_conf_get_int("performance_configuration_version_completed");
+  if(last_configure_version < DT_CURRENT_PERFORMANCE_CONFIGURE_VERSION)
+  {
+    // if this is the very first time -> run this configuration unconditionally
+    bool run_configure = TRUE;
+
+    // ask the user whether he/she would like
+    // dt to make changes in the settings
+    char *label_text = g_markup_printf_escaped(
```

Comment:
No need for `g_markup_printf_escaped`, you don't use markup in the string. Just pass `_("...")` to `dt_gui_show_standalone_yes_no_dialog()` instead of `label_text`.